### PR TITLE
Run commons checks and tests on Travis

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +28,20 @@ install:
...
+    - sudo apt-get autoremove postgis*
+    - sudo apt-get autoremove postgresql*
+    - sudo apt-get install postgresql-9.3-postgis-2.1
+    before_script:
+    - psql -c "CREATE USER \"www-data\" WITH PASSWORD 'www-data';" -U postgres
+    - psql -c "CREATE DATABASE c2cgeoportal_tests WITH OWNER c2cgeoform;" -U postgres
+    - psql -d c2cgeoportal_tests -c "CREATE EXTENSION postgis;" -U postgres
+    script:
+    - cd commons
+    - make check test
```
Comment (sbrunner) [inline .travis.yml]: This is a good first point but is should use the same test infra as for geoportal, is it OK for you to merge like it and I will uniformise this?

File: commons/c2cgeoportal_commons/__init__.py [hunk 2]
```diff
@@ -1,6 +1,12 @@
 """c2cgeoportal_commons package."""
 
-from c2cgeoportal_commons.models import get_session_factory, get_engine, get_tm_session, generate_mappers, init_dbsessions
+from c2cgeoportal_commons.models import (  # noqa
```
Comment (sbrunner) [inline commons/c2cgeoportal_commons/__init__.py]: Why noqa?

Comment (arnaud-morvan) [reply commons/c2cgeoportal_commons/__init__.py]: Why do you add this here ? it is unused, so it do not pass checks with noqa

Comment (arnaud-morvan) [reply commons/c2cgeoportal_commons/__init__.py]: I've supposed you add this imports here for use in c2cgeoportal, as I do not want to break c2cgeoportal, I've prefered to put noqa than to break something in c2cgeoportal.

File: .travis.yml (hunk 1 above)
Comment (arnaud-morvan) [reply .travis.yml]: Yes, I'd like to merge this ASAP.

Comment (arnaud-morvan) [reply .travis.yml]: Note we are not yet ready for docker and apache.

File: commons/c2cgeoportal_commons/__init__.py (hunk 2 above)
Comment (sbrunner) [reply commons/c2cgeoportal_commons/__init__.py]: No, they should be removed :-)

File: commons/README.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ make install
 ```
 sudo -u postgres psql -c "CREATE USER \"www-data\" WITH PASSWORD 'www-data';"
 
-DATABASE=c2cgeoportal_tests
+DATABASE=c2cgeoportal_test
```
Comment (sbrunner) [inline commons/README.md]: Just one test?

Comment (arnaud-morvan) [reply commons/README.md]: To be consistent with admin part, is this really relevant ?

Comment (sbrunner) [reply commons/README.md]: no :-)

File: commons/Makefile [hunk 4]
```diff
@@ -24,7 +24,7 @@ flake8: ${BUILD_DIR}/requirements-dev.timestamp
 
 ${BUILD_DIR}/venv.timestamp:
 	#Create a Python virtual environment.
-	python3 -m venv ${VENV}
+	virtualenv -p python3 ${VENV}
```
Comment (sbrunner) [inline commons/Makefile]: Why this change?

**Comment by arnaud-morvan** [comment]
Because the old version works locally but not on travis, I do not know exactly why.
